source/tools/testConsistency/ClientApp/app.js

```diff
@@ -0,0 +1,120 @@
+ // Copyright (c) Microsoft Corporation. All rights reserved. 
+ // Licensed under the MIT License. 
+const express = require("express");
+const path = require('path');
+const bodyParser = require('body-parser');
+const querystring = require('querystring');
+const optionsArray = require("./options");
+const postRequest = require("./postRequest");
+
+const getPostRequestPromise = postRequest.getPostRequestPromise;
+
+const app = express();
```

Comment:
somewhere there should be a class comment explaining what exactly it is that this "app" does; Ideally we'd also include a readme.md